tjeaneric/Flashcards-Frontend
# fetched data from graphql api

#### What does this PR do?
- Integration of Frontend and data from GraphQL API
#### Description of Task to be completed?
- Integration of Frontend and data from GraphQL API
#### How should this be manually tested?
- * clone the repo using SSH: `git clone git@github.com:tjeaneric/Flashcards-Frontend.git`
                          or    HTTPS: `git clone https://github.com/tjeaneric/Flashcards-Frontend.git`
- run `git checkout ft-integration-graphql-api`
- then `npm install`
- run `npm run dev`
- then check the route for the homepage  `localhost:{port}`
